### ISSUE TITLE
When Nautobot Notes Are Implemented, Changes May Be Needed

### ISSUE THREAD

**Issue by jvanderaa** [nautobot/nautobot-app-circuit-maintenance#190]
### Environment

- Current

### Proposed Functionality

Noticed that we have "Notes" as part of the current model. It looks like a global implementation may be upcoming. If this is implemented in Nautobot, we should remove/change the naming to avoid confusion - https://github.com/nautobot/nautobot/pull/1984

### Use Case

Prevent confusion of models

**Comment by chadell** [comment]
Totally, this was the first thought I got when I saw the new `notes` feature for Nautobot 1.4.
I think that there is too much duplication, so I would tend to remove the plugin one.

**Comment by scetron** [comment]
Agreed, will investigate this soon as 1.4 was released today on 16 Aug. `22

**Comment by glennmatthews** [comment]
This also causes some warnings during startup of Nautobot 2.0 with this app installed:

```
 ?: (drf_spectacular.W001) /opt/nautobot/.local/lib/python3.11/site-packages/nautobot_circuit_maintenance/api/serializers.py: Warning [MaintenanceNoteTaskView > NoteSerializer]: Encountered 2 components with identical names "Note" and different classes <class 'nautobot_circuit_maintenance.api.serializers.NoteSerializer'> and <class 'nautobot.extras.api.serializers.NoteSerializer'>. This will very likely result in an incorrect schema. Try renaming one.
 ?: (drf_spectacular.W001) /opt/nautobot/.local/lib/python3.11/site-packages/nautobot_circuit_maintenance/api/serializers.py: Warning [MaintenanceNoteTaskView > NoteSerializer]: Encountered 2 components with identical names "NoteRequest" and different classes <class 'nautobot_circuit_maintenance.api.serializers.NoteSerializer'> and <class 'nautobot.extras.api.serializers.NoteSerializer'>. This will very likely result in an incorrect schema. Try renaming one.
 ?: (drf_spectacular.W001) /opt/nautobot/.local/lib/python3.11/site-packages/nautobot_circuit_maintenance/api/serializers.py: Warning [MaintenanceNoteTaskView > NoteSerializer]: Encountered 2 components with identical names "PatchedNoteRequest" and different classes <class 'nautobot_circuit_maintenance.api.serializers.NoteSerializer'> and <class 'nautobot.extras.api.serializers.NoteSerializer'>. This will very likely result in an incorrect schema. Try renaming one.
 ?: (drf_spectacular.W001) /opt/nautobot/.local/lib/python3.11/site-packages/nautobot_circuit_maintenance/api/views.py: Warning [MaintenanceNoteTaskView]: Encountered 2 components with identical names "PaginatedNoteList" and different classes <class 'nautobot_circuit_maintenance.api.serializers.NoteSerializer'> and <class 'nautobot.extras.api.serializers.NoteSerializer'>. This will very likely result in an incorrect schema. Try renaming one.
```

**Comment by chadell** [comment]
fixed in #324
